src/main/resources/swagger/api-docs.yaml

```diff
@@ -614,6 +615,11 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Instance'
+      dateAccessed:
+        type: string
+        description: |
+          The date and time the cluster was last accessed, in ISO-8601 format.Date accessed is defined as the latest date
```

Comment:
Spacing is still off here - there’s a missing space after a period and an extra space before ‘when’ (swagger is nitpicky like that).

Also I feel like the wording could be clearer: “Date accessed is defined as the last time the cluster was created, modified, or accessed via the proxy.”